src/kernels/StressDivergenceDamper.C

```diff
@@ -33,8 +33,8 @@ StressDivergenceDamper::StressDivergenceDamper(const InputParameters & parameter
     _component(getParam<unsigned int>("component")),
     _ndisp(coupledComponents("displacements")),
     _disp_var(_ndisp),
-    _Fg(getMaterialPropertyByName<ColumnMajorMatrix>("global_forces")),
-    _Kg(getMaterialPropertyByName<ColumnMajorMatrix>("global_stiffness_matrix"))
+    _Fg(getMaterialPropertyByName<ColumnMajorMatrix>("fvd_global_forces")),
```

Comment:
You are changing these names so that there is consistency across all line type materials? Like Spring, Isolator, etc.?